### DISCUSSION TITLE
Is it possible to install CapRover on a home server with Dynamic IP and DDNS?

### DISCUSSION THREAD

**Discussion by ghost** [caprover/caprover#1986] [Q&A]
Hi, I would like to install CapRover because of its ease of use on my home server. I have a connection that unfortunately does not provide me with a static IP 123.456.1.789 but an IP that changes every 3/4 hours, I would like to use caprover under a ddns like DuckDns. It's possible to do it?
I ask this because when I start "caprover serversetup" it asks me the IP of the machine where cap is installed.

**Comment by githubsaturn** [comment]
Not officially supported, but some people have successfully used Cloudflare tunnels for this purpose.
https://github.com/caprover/caprover/issues/1788#issuecomment-1703817324

But please keep in mind that you may face issues as it's an unsupported and untested set up.
